tracking/translations_parser/utils.py

```diff
@@ -263,4 +266,15 @@ def publish_group_logs_from_tasks(
             yaml.dump(config, config_file)
 
         parents = str(logs_folder.resolve()).strip().split("/")
-        WandB.publish_group_logs(parents, project, group, existing_runs=[])
+        WandB.publish_group_logs(
+            logs_parent_folder=parents,
+            project=project,
+            group=group,
+            existing_runs=[],
+        )
+
+
+def suffix_from_group(group: str) -> str:
+    # Generate a unique hash of 5 characters from the group name, no matter the algorithm here
+    hash_val = hashlib.md5(group.encode()).hexdigest()[:5]
```

Comment:
I see, I made the changes you mentioned except for the last point. This one is quite specific, maybe we can open a follow-up to parse the task group ID from W&B group name instead of directly using the group name.

@eu9ene if you are aware of a way to parse a taskcluster group ID from the name (regex, function from the `taskcluster` package…) it would help a lot !

Actually running a new test run for online publication:
* https://wandb.ai/moz-translations/ru-en/groups/ci_JFmeeWVwQtmTNwAMDFwEQQ/workspace (from Taskcluster group [JFmeeWVwQtmTNwAMDFwEQQ](https://firefox-ci-tc.services.mozilla.com/tasks/groups/JFmeeWVwQtmTNwAMDFwEQQ))

This one is an old test run from CI, and can be removed from W&B:
* https://wandb.ai/moz-translations/ru-en/groups/ci_ZOmqhVQkQbaaOu6iOYZCIg/workspace

Reply:
ok, that sounds good. It looks fine on the dashboard. Please revert parameters.py and file an issue for whatever is left for the follow-up work, and then we can merge.